### PR TITLE
Bookstore: Add reducers and actions

### PR DESCRIPTION
**Pull request by fed1k**
# Hi Code Reviewer🙋‍♂️ I followed the requirements below:
  ✔ Open separate folders for separate files ( Books and Categories )
  ✔ Use reducers in Books and Categories files
  ✔ Use Actions in Books and Categories files
  ✔Export Actions and Reducers
  ✔ Use store in configureStore file and import all actions and reducer functions
  
  ## Happy Reviewing🐱‍👤

### CODE REVIEW

File: src/components/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const reducer = (initialState = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
```
Comment (elisha2kyakpo1) [inline src/components/redux/books/books.js]: - [x] This reducer should work as expected, unfortunately, when I tried to add or delete a book I failed to do that. Depending on the action type a user should be able to delete or add a book.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @fed1k,

You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight
- The project has no linter errors 👍
- The PR title and description looks professional ✔️


## Required Changes ♻️
_Check the comments under the review._
- [x] kindly make changes to the folder structure, let's remove redux folder from components folder.
```
/src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
```
- [x] Let's update the readme file and include commands required to run the project on the user's local machine. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @fed1k, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fed1k** [comment]
Thanks @BrianSammit

**Pull request merged**